attributioncode/validator.go

```diff
@@ -66,13 +64,6 @@ func (v *Validator) Validate(code, sig string) (string, error) {
 		}
 	}
 
-	if since, err := v.validateTimestamp(vals.Get("timestamp")); err != nil {
-		if since > 0 {
-			logEntry = logEntry.WithField("timestamp_age", since.Seconds())
-		}
-		logEntry.WithError(err).WithField("code_ts", vals.Get("timestamp")).Error("could not validate timestamp")
-		return "", err
-	}
 	vals.Del("timestamp")
```

Comment:
Will this throw an error if the `timestamp` key isn't there?

Reply:
Nope, [it](https://golang.org/pkg/net/url/#Values) uses [delete](https://golang.org/pkg/builtin/#delete) which should just be a no-op if that key doesn't exist.